radiant/sjra-62/dbt/models/staging/stg_locus_lookup.sql

```diff
@@ -0,0 +1,32 @@
+{{
+    config(
+        tags=['stg_locus_locup'],
+
+        materialized='table',
+        table_type='PRIMARY',
+        keys=['locus'],
+        distributed_by=['locus'],
+        buckets='5 ',
+
+        engine='OLAP',
+        properties="{'compression': 'LZ4', 'replication_num': '3', 'colocate_with': 'group_locus_id5'}",
+
+        pre_hook=["SET SESSION query_timeout = 10800;"]
+    )
+}}
+
+with locuses as (
+
+    select distinct {{ generate_surrogate_key(['chromosome', 'start', 'reference', 'alternate']) }} as locus
+    from {{ source('cqdg_datalake', 'normalized_snv') }}
+),
+
+combined as (
+    select
+        l.locus as locus,
+        uuid_numeric() as locus_id
```

Comment:
This is different from the `BIGINT AUTO_INCREMENT` implementation. 
The [`uuid_numeric`](https://docs.starrocks.io/docs/sql-reference/sql-functions/utility-functions/uuid_numeric) creates a `LARGEINT` (16 bytes). 

This enables the creation of `stg_locus_lookup` through `dbt`.

(Generated the whole bunch of `uuid` in about ~5 minutes)